src/main/java/io/orten/nano/model/Organization.java

```diff
@@ -1,37 +1,38 @@
 package io.orten.nano.model;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import org.hibernate.annotations.DynamicUpdate;
+import org.hibernate.annotations.SelectBeforeUpdate;
+
+import javax.persistence.*;
+import java.io.Serializable;
 /*
   represents an organization raising fund for one or more projects
```

Comment:
change to a javadoc syntax, i.e.
/**
   \* represents an organzation...
   */

Reply:
added the comment style as suggested

Reply:
done